### PR TITLE
✨ Show timestamps in webvtt format

### PR DESCRIPTION
**Pull request by pajowu**

### CODE REVIEW

File: frontend/src/editor/transcription_editor.tsx [hunk 1]
```diff
@@ -16,6 +16,18 @@ const LazyDebugPanel = lazy(() =>
   import('./debug_panel').then((module) => ({ default: module.DebugPanel })),
 );
 
+function formatTimestamp(time: number): string {
+  const seconds = time % 60;
+  time = (time - seconds) / 60;
+  const minutes = time % 60;
+  const hours = (time - minutes) / 60;
+  if (hours > 0) {
```
Comment (rroohhh) [inline frontend/src/editor/transcription_editor.tsx:24]: I don't know what the convention is, but maybe this check should take the total length into account?

**Review by rroohhh — APPROVED** [review]

File: frontend/src/editor/transcription_editor.tsx (hunk 1 above)
Comment (pajowu) [reply frontend/src/editor/transcription_editor.tsx]: Ah wait, you're right. This should check the total length of the audio, not just the current timestamp.

Comment (pajowu) [reply frontend/src/editor/transcription_editor.tsx]: Once the `reencode_worker` branch is landed, we can add this. At the moment, this isn't easily possible

Comment (pajowu) [reply frontend/src/editor/transcription_editor.tsx]: Once the `pajowu/stats` branch (#144) is merged, we can easily to that

**Pull request merged**